main.js

```diff
@@ -0,0 +1,29 @@
+
+var $h1 = document.createElement('h1')
+$h1.textContent = 'Typing Tutor'
+document.body.appendChild($h1)
+var string = 'grumpy wizard make toxic brew for the evil queen and jack'
+var charsArray = string.split('')
+var objArray = []
+for (var i = 0; i < charsArray.length; i++) {
+  var obj = {}
+  obj.letter = charsArray[i]
+  objArray.push(obj)
+}
+//console.log(objArray)
```

Comment:
Please remove all `console.log` and commented code before submitting a Pull Request.